### PR TITLE
Add Tests for the methods in ReminderOnOff class

### PR DESCRIPTION
**Pull request by jasperng-nus**
Added test cases for the ReminderOnOff class.

### CODE REVIEW

File: src/test/java/seedu/address/model/reminder/ReminderOnOffTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        assertFalse(ReminderOnOff.isValidSetReminder("")); // empty string
+        assertFalse(ReminderOnOff.isValidSetReminder(" ")); // spaces only
+        assertFalse(ReminderOnOff.isValidSetReminder("truee")); // incorrect spelling
+        assertFalse(ReminderOnOff.isValidSetReminder("falsee")); // incorrect spelling
+        assertFalse(ReminderOnOff.isValidSetReminder("phone")); // non-boolean
+        assertFalse(ReminderOnOff.isValidSetReminder("1")); // non-boolean
+
+        // valid set reminder
+        assertTrue(ReminderOnOff.isValidSetReminder("true")); // true
+        assertTrue(ReminderOnOff.isValidSetReminder("false")); // false
```
Comment (AjayArvind2207) [inline src/test/java/seedu/address/model/reminder/ReminderOnOffTest.java:48]: You can also consider adding different capitalizations to show that it is case insensitive

**Review by AjayArvind2207 — COMMENTED** [review]
LGTM other than one minor nit 👍

**Comment by AjayArvind2207** [comment]
LGTM!

**Comment by Xilef121** [comment]
LGTM

**Pull request merged**